src/clj/datasplash/bq.clj

```diff
@@ -22,16 +22,26 @@
    (org.apache.beam.sdk.values PBegin PCollection)))
 
 (defn read-bq-raw
-  [{:keys [query table standard-sql? query-location] :as options} p]
+  [{:keys [query table standard-sql? query-location temp-project temp-dataset] :as options} p]
   (let [opts (assoc options :label :read-bq-table-raw)
         ptrans (cond
                  query (cond-> (.fromQuery (BigQueryIO/readTableRows) query)
-                         standard-sql? (.usingStandardSql)
-                         query-location (.withQueryLocation query-location))
+                               standard-sql? (.usingStandardSql)
```

Comment:
Should not be changed, may conflict with cljfmt check.

Reply:
fixed, cljfmt check ok locally, actions seems broken (lein: command not found)